build.sbt

```diff
@@ -3189,10 +3191,17 @@ lazy val `runtime-version-manager` = project
   .dependsOn(pkg)
   .dependsOn(downloader)
   .dependsOn(cli)
+  .dependsOn(`process-utils`)
   .dependsOn(`version-output`)
   .dependsOn(`edition-updater`)
   .dependsOn(`distribution-manager`)
 
+lazy val `process-utils` = project
```

Comment:
I am starting to like documentation comments on top of the project describing its purpose.